tests/PySys/pysysproject.xml

```diff
@@ -7,7 +7,7 @@
 	<property name="tebasedir" value="${env.TEBASEDIR}" default="${testRootDir}/../.."/>
 	<property name="exampledir" value="${env.EXAMPLEDIR}" default="${testRootDir}/../../ci"/>
 	<property name="c8yurl" value="${env.C8YURL}" default=""/>
-	<property name="username" value="${env.C8YUSERNAME}" default=""/>
+	<property name="c8yusername" value="${env.C8YUSERNAME}" default=""/>
```

Comment:
I wish it was `c8y_username` but I see that you've just kept it consistent with the rest of the names.

Reply:
There is always room for renaming all the vars.